### ISSUE TITLE
Tankmouse | Mousescrolling doesn't work anymore

### ISSUE THREAD

**Issue by StevenHachel** [SukkoPera/OpenAmigaJoyMouseSwitcher#6]
Hello,

If this adapter is connected, scrolling with the tank mouse no longer works. Are any signals filtered out?

Steff

**Comment by SukkoPera** [comment]
Good question. Any idea how the scrolling protocol works? What signals does it use?

**Comment by StevenHachel** [comment]
Thanks for the quick reply.
Unfortunately I can't answer that for you.
I would have to check to see which signals are not being allowed through or cannot be processed.

Steven

**Comment by StevenHachel** [comment]
Using the Rawkey (Aminet) tool I get the following values for the mouse wheel:
WheelBac CODE $007B QUAL $0000
WheelFor CODE $007A QUAL $0000

**Comment by SukkoPera** [comment]
Those are probably "synthetic" software events that get generated when the mouse sends the scroll events. How the latter takes place at the electrical level is what we need to figure out. All pins get through (for pin 5 you need to move the jumper), but the pins are monodirectional when the switcher is in the path. If the Amiga needs to send signals to the mouse to get scrolling events, it won't work (and it's actually dangerous). It's the same reason why CD32 controllers don't work.

**Comment by StevenHachel** [comment]
Okay, thanks for the detailed answer. Then I'll have to live with it and it's not a problem for me. At least now I know.

Greetings and thanks 
Steven

**Comment by SukkoPera** [comment]
Let me reopen this. It would be good if anyone could help with the scrolling protocol, at least to understand whether the switcher just prevents that from working or if any hardware is put at risk.

**Comment by StevenHachel** [comment]
Okay, that makes sense. 😊

**Comment by StevenHachel** [comment]
Is there any news regarding this?

**Comment by SukkoPera** [comment]
Nope, no one has provided any information so far and I'm not actively working on it.
